Source/SwiftLintFramework/Rules/Lint/CommentSpacingRule.swift

```diff
@@ -0,0 +1,143 @@
+import Foundation
+import SourceKittenFramework
+
+public struct CommentSpacingRule: OptInRule, ConfigurationProviderRule, SubstitutionCorrectableRule, AutomaticTestableRule {
```

Comment:
This line is over our column limit of 120 characters, which is why almost all of the CI jobs are failing.

Reply:
Oops, thanks for calling this out!